.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -209,6 +209,11 @@ jobs:
               # clusterbuster: disable prometheus logs
               ENABLE_PROMETHEUS_SNAPSHOT='False'
             fi
+            if [[ '${{ matrix.workload }}' == 'vdbench_vm' ]]
+            then
+              # Warm-up: download vdbench_vm container disk image per node (centos stream8)
+              ssh -t provision "podman run --rm -t -e WORKLOAD='vdbench_vm_scale' -e KUBEADMIN_PASSWORD='$KUBEADMIN_PASSWORD' -e SCALE='$SCALE' -e SCALE_NODES=$SCALE_NODES -e REDIS='$REDIS' -e RUN_TYPE='test_ci' -e SAVE_ARTIFACTS_LOCAL='False' -e THREADS_LIMIT='$THREADS_LIMIT' -e TIMEOUT='$TIMEOUT' -e log_level='INFO' -v '$CONTAINER_KUBECONFIG_PATH':'$CONTAINER_KUBECONFIG_PATH' --privileged 'quay.io/ebattat/benchmark-runner:latest'"
```

Comment:
Simply do

```
ssh provision -- podman pull quay.io/ebattat/benchmark-runner:latest
```
You don't need to actually run anything, just pull the image.

Reply:
We must run it on scale because I want to download it in every worker node

Reply:
Oh I see, you want to run it on every node.  You might want to add a dummy workload that simply exits without running anything; you could probably even do

```
WORKLOAD='no_workload'
```
That run would of course fail, but it would achieve the purpose of pulling the image.